federatedscope/core/trainers/base_trainer.py

```diff
@@ -0,0 +1,29 @@
+import abc
+
+
+class BaseTrainer(abc.ABC):
+    def __init__(self, model, data, device, **kwargs):
+        self.model = model
+        self.data = data
+        self.device = device
+        self.kwargs = kwargs
+
+    @abc.abstractmethod
+    def train(self):
```

Comment:
do we need a abstract method for fintuning?

Reply:
I think finetuning is optional, and if someone needs ft, he could implement finetuning in his way.